core-linux/bin/app/settings.json

```diff
@@ -5,6 +5,8 @@
     "window" : {
         "width" : "1000",
         "height" : "700",
-        "fullscreen" : false
+        "fullscreen" : false,
+        "alwaysOnTop" : false,
```

Comment:
Could you simply use `alwaystop` or `alwaysontop` matching lowercase style of other settings

Reply:
sure